pkg/store/artifactory_store.go

```diff
@@ -112,35 +116,59 @@ func NewArtifactoryStore(options ArtifactoryStoreOptions) (Store, error) {
 
 func (s *ArtifactoryStore) getKey(stack string, component string, key string) (string, error) {
 	if s.stackDelimiter == nil {
-		return "", fmt.Errorf("stack delimiter is not set")
+		return "", ErrStackDelimiterNotSet
 	}
 
 	prefixParts := []string{s.repoName, s.prefix}
 	prefix := strings.Join(prefixParts, "/")
+
 	return getKey(prefix, *s.stackDelimiter, stack, component, key, "/")
 }
 
-func (s *ArtifactoryStore) Get(stack string, component string, key string) (interface{}, error) {
+func (s *ArtifactoryStore) validateGetParams(stack, component, key string) error {
 	if stack == "" {
-		return nil, fmt.Errorf("stack cannot be empty")
+		return ErrEmptyStack
 	}
 
 	if component == "" {
-		return nil, fmt.Errorf("component cannot be empty")
+		return ErrEmptyComponent
 	}
 
 	if key == "" {
-		return nil, fmt.Errorf("key cannot be empty")
+		return ErrEmptyKey
+	}
+
+	return nil
+}
+
+func (s *ArtifactoryStore) processDownloadedFile(tempDir, paramName string) (interface{}, error) {
```

Comment:
@mcalhoun we can remove `gofmt` from the `.golangci-lint.yaml` config, because it's not identical to `gofumpt`